applications/OptimizationApplication/python_scripts/helmholtz_analysis.py

```diff
@@ -0,0 +1,47 @@
+
+from sys import argv
+
+import KratosMultiphysics as Kratos
+
+from KratosMultiphysics.analysis_stage import AnalysisStage
+from KratosMultiphysics.OptimizationApplication.solvers.helmholtz_solver import HelmholtzSolver
+
+class HelmholtzAnalysis(AnalysisStage):
+    def _CreateSolver(self):
+        return HelmholtzSolver(self.model, self.project_parameters["solver_settings"])
+
+    def _GetOrderOfProcessesInitialization(self):
+        # The list of processes will contain a list with each individual process already constructed (boundary conditions, initial conditions and gravity)
+        # Note 1: gravity is constructed first. Outlet process might need its information.
+        # Note 2: initial conditions are constructed before BCs. Otherwise, they may overwrite the BCs information.
+        return ["initial_conditions_process_list",
+                "boundary_conditions_process_list",
+                "auxiliar_process_list"]
```

Comment:
```suggestion
                "auxiliary_process_list"]
```
Typo

Reply:
This is a known typo in Kratos, and unless it is changed from the core, we cannot change it.